pyanaconda/modules/common/structures/storage.py

```diff
@@ -453,3 +453,61 @@ def get_root_device(self):
         :return: a device name or None
         """
         return self.mount_points.get("/")
+
+
+class PlatformMountPointData(DBusData):
+    """Platform required mount points data"""
+
+    def __init__(self):
+        self._value = ""
+        self._filesystem = ""
+        self._encrypted = False
+        self._lv = False
+
+    @property
+    def value(self) -> Str:
```

Comment:
It would be good to keep the DBus structures consistent. So `value` should be probably `mount_point` and `fstype` should be `file_system_type`....